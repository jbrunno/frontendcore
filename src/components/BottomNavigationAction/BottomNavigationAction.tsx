import React from 'react';
import {
  BottomNavigationAction as MuiBottomNavigationAction,
  BottomNavigationActionProps as MuiBottomNavigationActionProps,
} from '@mui/material';

export type BottomNavigationActionProps = MuiBottomNavigationActionProps;

export function BottomNavigationAction(props: BottomNavigationActionProps) {
  return <MuiBottomNavigationAction {...props} />;
}
export default BottomNavigationAction;
