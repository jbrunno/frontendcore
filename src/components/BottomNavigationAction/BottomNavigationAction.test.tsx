import React from 'react';
import { render } from '@test-utils/component-testing';

import { BottomNavigationAction } from './BottomNavigationAction';

test('renders the BottomNavigationAction', () => {
  const { container } = render(<BottomNavigationAction />);
  expect(container).toMatchSnapshot();
});
