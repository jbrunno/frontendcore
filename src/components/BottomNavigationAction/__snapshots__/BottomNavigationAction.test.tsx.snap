// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the BottomNavigationAction 1`] = `
.emotion-0 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
  position: relative;
  box-sizing: border-box;
  -webkit-tap-highlight-color: transparent;
  background-color: transparent;
  outline: 0;
  border: 0;
  margin: 0;
  border-radius: 0;
  padding: 0;
  cursor: pointer;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  vertical-align: middle;
  -moz-appearance: none;
  -webkit-appearance: none;
  -webkit-text-decoration: none;
  text-decoration: none;
  color: inherit;
  -webkit-transition: color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,padding-top 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  transition: color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,padding-top 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  padding: 0px 12px;
  min-width: 80px;
  max-width: 168px;
  color: #3B3B4199;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  padding-top: 0;
}

.emotion-0::-moz-focus-inner {
  border-style: none;
}

.emotion-0.Mui-disabled {
  pointer-events: none;
  cursor: default;
}

@media print {
  .emotion-0 {
    -webkit-print-color-adjust: exact;
    color-adjust: exact;
  }
}

.emotion-0.Mui-selected {
  color: #4A31FF;
}

.emotion-1 {
  font-family: Lato,sans-serif;
  font-size: 0.75rem;
  opacity: 0;
  -webkit-transition: font-size 0.2s,opacity 0.2s;
  transition: font-size 0.2s,opacity 0.2s;
  transition-delay: 0s;
}

.emotion-1.Mui-selected {
  font-size: 0.875rem;
}

.emotion-2 {
  overflow: hidden;
  pointer-events: none;
  position: absolute;
  z-index: 0;
  top: 0;
  right: 0;
  bottom: 0;
  left: 0;
  border-radius: inherit;
}

<div>
  <button
    class="MuiButtonBase-root MuiBottomNavigationAction-root MuiBottomNavigationAction-iconOnly emotion-0"
    tabindex="0"
    type="button"
  >
    <span
      class="MuiBottomNavigationAction-label MuiBottomNavigationAction-iconOnly emotion-1"
    />
    <span
      class="MuiTouchRipple-root emotion-2"
    />
  </button>
</div>
`;
